config/output_schemas/gb_sas_schema.toml

```diff
@@ -3,7 +3,7 @@ old_name = "reference"
 Deduced_Data_Type = "Int64"
 
 [period]
-old_name = "period"
+old_name = "period_year"
```

Comment:
It's nice that we can choose what column goes into output.
The risk is, the toml files are not part of the package, so they need to be updated manually of the v_0.7.0 of the runner.